api/audit/permissions.py

```diff
@@ -15,3 +16,15 @@ def has_permission(self, request: Request, view: View):
             return False
 
         return request.user.has_organisation_permission(organisation, VIEW_AUDIT_LOG)
```

Comment:
Should this permission class be removed? Since there is no VIEW_AUDIT_LOG at organisation level?

Reply:
@matthewelwell I understood we want to keep the permission view at organisation level, is that correct?

Reply:
@novakzaballa  can you add a comment here explaining why are we keeping it

Reply:
After discussing this with @matthewelwell, I am changing it to only allow organisation admins.